<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWizard</class>
 <widget class="QDialog" name="SetupWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Wizard - MEGAsync</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#sPages
{
	background-image: url(://images/background.png);
}

#lVerify
{
	color: red;
}

#rTypicalSetup
{
	padding: 15px;
	border-radius: 10px;
}

#rAdvancedSetup
{
	padding: 15px;
	border-radius: 10px;
}

#rTypicalSetup:checked
{
	background-color: rgb(242,242,242);
}

#rAdvancedSetup:checked
{
	background-color: rgb(242,242,242);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sPages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pSetup">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lSetupHeader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>://images/logo.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wHaveAccount" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>140</number>
          </property>
          <property name="topMargin">
           <number>55</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rHaveAccount">
            <property name="text">
             <string>I have a Mega account</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rDontHaveAccount">
            <property name="text">
             <string>I don't have a Mega account</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vsSetup">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pNewAccount">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>100</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QLabel" name="lCreateAccountHeader">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>://images/app_128.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lCreateAccount">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create a new Mega account</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wCreateAccountForm" native="true">
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="eName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lEmail">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="eEmail"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ePassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lRepeatPassword">
            <property name="text">
             <string>Repeat password:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="eRepeatPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="hAcceptTerms">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="cAgreeWithTerms">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lTermsLink">
              <property name="text">
               <string>I agree with the MEGA &lt;a href=&quot;https://mega.co.nz/#terms&quot;&gt;Terms of Service&lt;/a&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hsTerms">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsNewAccount">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pLogin">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>35</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="lLoginHeader">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>://images/app_128.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lLogin">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Login to your MEGA account</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wLoginForm" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="leftMargin">
           <number>140</number>
          </property>
          <property name="topMargin">
           <number>35</number>
          </property>
          <property name="rightMargin">
           <number>140</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lLoginEmail">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="eLoginEmail"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lLoginPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="eLoginPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsLogin">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lVerify">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Please verify your account using the confirmation link that we have sent to your email account</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pProgress">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="leftMargin">
           <number>80</number>
          </property>
          <property name="topMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>80</number>
          </property>
          <property name="bottomMargin">
           <number>100</number>
          </property>
          <item>
           <widget class="QLabel" name="lProgress">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Logging in ...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="value">
             <number>-1</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pSetupType">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="lSetupType">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Choose install type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rTypicalSetup">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Typical</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>://images/app_48.png</normaloff>://images/app_48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rAdvancedSetup">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Advanced</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>://images/app_48.png</normaloff>://images/app_48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vsSetupType">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pAdvanced">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="lAdvancedSetup">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Advanced configuration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>The following folders will be automatically synchronized:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wAdvancedConfigForm" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lLocalFolder">
            <property name="text">
             <string>Local folder:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lMegaFolder">
            <property name="text">
             <string>MEGA folder:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="hMegaFolder">
            <item>
             <widget class="QLineEdit" name="eLocalFolder">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bLocalFolder">
              <property name="text">
               <string>Change</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="hLocalFolder">
            <item>
             <widget class="QLineEdit" name="eMegaFolder">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bMegaFolder">
              <property name="text">
               <string>Change</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="vsAdvancedSetup">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pWelcome">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>80</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>80</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="lWelcome">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Welcome to MEGA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lWelcomeImage">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>://images/app_128.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Your local folder:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lFinalLocalFolder">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>and your MEGA folder:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lFinalMegaFolder">
         <property name="font">
          <font>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>will be automatically synchronized.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wButtons" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="hsButtons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bBack">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bNext</tabstop>
  <tabstop>bBack</tabstop>
  <tabstop>bCancel</tabstop>
  <tabstop>rHaveAccount</tabstop>
  <tabstop>rDontHaveAccount</tabstop>
  <tabstop>eName</tabstop>
  <tabstop>eEmail</tabstop>
  <tabstop>ePassword</tabstop>
  <tabstop>eRepeatPassword</tabstop>
  <tabstop>cAgreeWithTerms</tabstop>
  <tabstop>eLoginEmail</tabstop>
  <tabstop>eLoginPassword</tabstop>
  <tabstop>rTypicalSetup</tabstop>
  <tabstop>rAdvancedSetup</tabstop>
  <tabstop>eLocalFolder</tabstop>
  <tabstop>bLocalFolder</tabstop>
  <tabstop>eMegaFolder</tabstop>
  <tabstop>bMegaFolder</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
